lib/lc/lc.c

```diff
@@ -105,7 +105,7 @@ static struct kmem_cache *lc_cursor_cache;
  */
 struct ingest_batch {
     u64                  ib_seqno;
-    atomic64_t           ib_refcnt;
+    atomic_int           ib_refcnt;
```

Comment:
Should this be a 64-bit atomic type?

Reply:
Lemme review it again to be sure..

Reply:
The comment says it's the number of cursors with some given horizon.  Given we can't create more than 20,000 or so cursors I think atomic_int should suffice 😄 @gsramdasi